### ISSUE TITLE
Redirect Request

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#57819]
### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [ ] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
Now that we need to have more than 1 year of past education rates, we needed to change the url of the page we already had live.

### Implementation date
When does this request need to be live?
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/education/benefit-rates/montgomery-selected-reserve-rates/past-rates/ | https://www.va.gov/education/benefit-rates/montgomery-selected-reserve-rates/past-rates-2021-2022/
https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates/ | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2021-2022/
https://www.va.gov/education/benefit-rates/chapter-35-rates/past-rates/ | https://www.va.gov/education/benefit-rates/chapter-35-rates/past-rates-2021-2022/
https://www.va.gov/education/benefit-rates/montgomery-active-duty-rates/past-rates/ | https://www.va.gov/education/benefit-rates/montgomery-active-duty-rates/past-rates-2021-2022/
https://www.va.gov/education/benefit-rates/post-9-11-gi-bill-rates/past-rates/ | https://www.va.gov/education/benefit-rates/post-9-11-gi-bill-rates/past-rates-2021-2022/

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by RLHecht** [comment]
@jennymayoco these are ready for you to vet.

**Comment by jennymayoco** [comment]
@RLHecht - I'm getting an error when I click on any of the current URL pages. Confirming if this is correct?

**Comment by RLHecht** [comment]
Yep, that's right. We changed the urls of the 2021-2022 past rates when we added 2020-2021 rates (we weren't distinguishing between years). So we want to redirect the original urls in case anyone had them bookmarked.

**Comment by jennymayoco** [comment]
@RLHecht Thank you for clarifying!

@jilladams @wesrowe these redirects are vetted and ready. FYI the current URLs are already down.
